gensim/topic_coherence/indirect_confirmation_measure.py

```diff
@@ -35,30 +37,75 @@
 logger = logging.getLogger(__name__)
 
 
-def cosine_similarity(segmented_topics, accumulator, topics, measure='nlr', gamma=1):
+def word2vec_similarity(segmented_topics, accumulator, with_std=False):
+    """For each topic segmentation, compute average cosine similarity using a
+    WordVectorsAccumulator.
+
+    Args:
+    ----
+    segmented_topics : Output from the segmentation module of the segmented topics.
+                       Is a list of list of tuples.
+    accumulator : Output from the probability_estimation module.
+                  Is an accumulator of word occurrences (see text_analysis module).
+    with_std : True to also include standard deviation across topic segment sets in addition
+               to the mean coherence for each topic; default is False.
+
+    Returns:
+    -------
+    topic_coherences : list of word2vec cosine similarities per topic.
+    """
+    topic_coherences = []
+    for i, topic_segments in enumerate(segmented_topics):
+        segment_similarities = []
+        for w_prime, w_star in topic_segments:
+            if not hasattr(w_prime, '__iter__'):
+                w_prime = [w_prime]
+            if not hasattr(w_star, '__iter__'):
+                w_star = [w_star]
+
+            try:
+                segment_similarities.append(accumulator.ids_similarity(w_prime, w_star))
+            except ZeroDivisionError:
+                logger.warn("at least one topic word not in word2vec model")
+
+        if with_std:
+            topic_coherences.append((np.mean(segment_similarities), np.std(segment_similarities)))
+        else:
+            topic_coherences.append(np.mean(segment_similarities))
+
+    return topic_coherences
+
+
+def cosine_similarity(segmented_topics, accumulator, topics, measure='nlr', gamma=1,
+                      with_std=False):
```

Comment:
Don't use vertical indent (here you have not very long line) please make definition in one line

Reply:
I added a new argument here, which made it longer, so I used the hanging indent instead.